docs/docs/deploy/deploy-rasa-plus.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: deploy-rasa-plus
+sidebar_label: "Deploy Rasa Plus"
+title: "Deploy Rasa Plus in Production"
+description: Deploy Rasa Plus
+---
+import RasaProLabel from '@theme/RasaProLabel';
+import RasaProBanner from '@theme/RasaProBanner';
+
+<RasaProLabel />
+
+<RasaProBanner/>
+
+To use the Rasa Plus image in a Helm deployment, you will need to substitute `rasa` with `rasa-plus` as
```

Comment:
This feels a little abrupt to me. This assumes that you already have OSS deployed and are now just replacing it with the rasa-plus image. We should either provide documentation from scratch (nothing is deployed) or at least refer back the the OSS deployment so they know what is required. I have a slight preference for providing documentation from scratch so that users are not jumping back and forth

Reply:
discussed and on Nathans plate